example-get-started/README.md

```diff
@@ -0,0 +1,127 @@
+A set of scripts to generate an NLP DVC Studio project with multiple branches,
+commit history, experiments, metrics, plots, etc. It used in the DVC docs and in
+Studio as a demo project. 
+
+This script can be also used in an advanced scenario to generate a nested
+mono-repositories that are used as fixtures in Studio testing, or testing
+different remote types. See the section below for the advanced settings.
+
+## Demo project
+
+Note! In some cases, before rebuilding the project you might want to delete the
+existing remote tags if you change the order, or names.
+
+```shell
+git clone git@github.com:<slug>/example-get-started.git
+cd example-get-started
+git tag -l | xargs -n 1 git push --delete origin
+```
+
+For the basic use case (docs and Studio demo), use the command below.
+
+```shell
+./generate.sh
+```
+
+If change source code, to publish it on S3 (needed for the get started tutorial)
+pass `true` to the command. It's needed when you ready to publish it.
+
+```shell
+./generate.sh true
+```
+
+The repo generated in `build/example-get-started` is intended to be published on
+to the https://github.com/iterative/example-get-started. Make sure the Github
+repo exists first and that you have appropriate write permissions.
+
+To create it with https://cli.github.com/, run:
+
+```shell
+gh repo create iterative/example-get-started --public \
+     -d "Get Started DVC project" -h "https://dvc.org/doc/get-started"
+```
+
+Run these commands to force push it:
+
+```shell
+cd build/example-get-started
+git remote add origin git@github.com:<slug>/example-get-started.git
+git push --force origin main
+git push --force origin try-large-dataset
+git push --force origin tune-hyperparams
+# we push git tags one by one for Studio to receive webhooks:
+git tag --sort=creatordate | xargs -n 1 git push --force origin
```

Comment:
```suggestion
cd build/example-get-started
git remote add origin git@github.com:iterative/example-get-started.git
git remote add fixture git@github.com:iterative/example-get-started-fixture.git
git push --force origin main
git push --force fixture main
git push --force origin try-large-dataset
git push --force fixture try-large-dataset
git push --force origin tune-hyperparams
git push --force fixture tune-hyperparams
# we push git tags one by one for Studio to receive webhooks:
git tag --sort=creatordate | xargs -n 1 git push --force origin
git tag --sort=creatordate | xargs -n 1 git push --force fixture
```

Reply:
I guess we need some note here that after you push this, tests in Studio can be broken. Should probably be expected by everyone, but better to have a reminder you need to check them and fix if needed.

Reply:
Also, from my user perspective, I'd want to first push everything to example-get-started, then go check everything is correct (open View, MR, MDP), and then want to push to fixture repo. So IMHO it's more convenient to have this separated.

Reply:
> Also, from my user perspective, I'd want to first push everything to example-get-started, then go check everything is correct (open View, MR, MDP), and then want to push to fixture repo. So IMHO it's more convenient to have this separated.

To my mind, the "right" process would be that you test the things in a personal copy or something, then when you have verified that everything is ok, merge the P.R to `main` in this repo, and everything is automatically generated/pushed to the right repos

Reply:
We are missing the automation in `main` branch

Reply:
Right, but how you do that with codespaces? Are you able to push to a repo in a personal account?

Reply:
Though it probably can be resolved with running `gh auth`.

Reply:
folks, I think we are overcomplicating this ... we can mention that we also have fixtures (and in this case we'll have 5-6 of them - we can't in one README include all the details on how to generate them). Each fixture repo should include that in its readme (and that's what I do here)+ we should have a list of them somewhere (I'll start on that after I get through monorepo and MR tests - I hit bugs even writing tests, that's why it takes longer)